daemon/control.c

```diff
@@ -1,6 +1,6 @@
 /*
  * This file is part of trust|me
- * Copyright(c) 2013 - 2020 Fraunhofer AISEC
+ * Copyright(c) 2017 - 2020 Fraunhofer AISEC
```

Comment:
remove this change, you misinterpreted my personally given comment about the header
here. Fix up copyright header in control.proto instead,
from 2013 - 2017 -> 2013 - 2020

Reply:
Done